### PR TITLE
Add WORKFLOW_REPOS project

### PR DESCRIPTION
**Pull request by luator**
## Description

This project lists all repositories which should receive the common GitHub workflows.
Having this as a dedicated treep project makes it easier to run the script that copies the workflows to all of them.

For now this are all isr/mpi-is repos of the PAM_MUJOCO project plus the github_workflow_manager.

## How I Tested

By running `treep --clone WORKFLOW_REPOS`

### CODE REVIEW

File: projects.yaml [hunk 1]
```diff
@@ -27,4 +27,24 @@ PAM_MUJOCO:
 
 BALL_TRACKING:
     parent_projects: ['AMENT']
-    repos: ['tennicam_proto','tennicam','mpi_cmake_modules']
\ No newline at end of file
+    repos: ['tennicam_proto','tennicam','mpi_cmake_modules']
```
Comment (vincentberenz) [inline projects.yaml:30]: I would remove tennicam and tennicam_proto, they are a superficial refactor of legacy code that would take possibly too much time to make proper

File: projects.yaml [hunk 2]
```diff
@@ -27,4 +27,24 @@ PAM_MUJOCO:
...
+
+# List of repositories to which common GitHub workflows should be added (e.g.
+# for style checks on pull requests).
+WORKFLOW_REPOS:
+    repos: [
```
Comment (vincentberenz) [inline projects.yaml:35]: tennicam_client seems to be missing (was added only very recently)

**Review by vincentberenz — CHANGES_REQUESTED** [review]

File: projects.yaml (hunk 2 above)
Comment (luator) [reply projects.yaml:35]: added

File: projects.yaml (hunk 1 above)
Comment (luator) [reply projects.yaml:30]: I'm not sure I understand, they are currently not included in the WORKFLOW_REPOS project.  Or do you mean to remove them completely from treep?

Comment (vincentberenz) [reply projects.yaml:30]: my bad, went too fast, all good

**Review by vincentberenz — APPROVED** [review]

**Pull request merged**